src/ocean/util/app/DaemonApp.d

```diff
@@ -563,7 +564,16 @@ public abstract class DaemonApp : Application,
 
     private bool statsTimer ( )
     {
-        this.onStatsTimer();
+        try
+        {
+            this.onStatsTimer();
+        }
+        catch (Exception e)
+        {
+            Stderr.formatln("Exception caught in DaemonApp.statsTimer: {}",
+                    getMsg(e));
```

Comment:
I would really like a `toString` here (so just `e`) because this way we get a stack trace for free in D2.

Reply:
Wasn't the point that getting the stack trace is very slow and should be avoided?

Reply:
> If this is hit, it's arguably a bug.
  
Not sure about that. If you're out of disk quota, for example, this will happen.

Reply:
> Also it's once every 30s, not hundreds of occurences / second
  
That's a fair point.